.github/workflows/cd-chromatic.yml

```
@@ -0,0 +1,51 @@
+# .github/workflows/chromatic.yml
+
+# Workflow name
+name: '🌈 Chromatic'
+
+# Event for the workflow
+on: pull_request
```

Let's only trigger Chromatic builds for changes to the participant portal
```suggestion
on: 
  pull_request:
        paths:
          - participant_portal/**
```